samcli/commands/_utils/options.py

```diff
@@ -788,3 +788,23 @@ def _space_separated_list_func_type(value):
 
 
 _space_separated_list_func_type.__name__ = "LIST"
+
+
+def generate_next_command_recommendation(command_tuples: list):
```

Comment:
- Since we now have this new function, can we have some unit tests for it?
- we are expecting `command_tuples` to be a list of tuples, can we be more precise about the type hint?
- Can we also update other commands which give "next command suggestions" to use this function?

Reply:
Done